md/csrf-security.md

```diff
@@ -73,3 +73,17 @@ To activate the addition of the secure flag, edit the configuration file and cha
 [User authentication overview](user-authentication-overview.md)
 [Read more about CSRF attacks](http://www.acunetix.com/websitesecurity/csrf-attacks)
 
+
+## How to use UI form or page from 7.3.3 or lower to 7.4.0 or greater when CSRF security is enabled
```

Comment:
maybe rephrase the title, in a shorter one, or at least remove "from 7.3.3 or lower " :

"How to Migrate your pages to 7.4.0 or greater when CSRF security is enabled"